server/capabilities/gke_subnetworks_endpoint.go

```diff
@@ -0,0 +1,94 @@
+package capabilities
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"google.golang.org/api/compute/v1"
```

Comment:
Separate this import out from the other golang imports.

Reply:
Updated!